### PR TITLE
Update 1

### PR DESCRIPTION
**Pull request by Rainb0wKey**
The first five pages have been updated. 

###### There may be a lot of errors

### CODE REVIEW

File: src/bdscript/addEmoji.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ Adds an emoji to the server.
...
+$addEmoji[name;image URL;return emoji?]
 ```
 
-### Breakdown
-- `name` - The name of the new emoji.
-- `imageURL` - The image that the emoji showcases. This needs to be a valid image URL.
-- `returnEmoji` - Whether to show the emoji in the bot's message or not.
+## Parameters
+- `name`: The name of the new emoji.
+- `image URL`: The image that the emoji showcases. This needs to be a valid image URL.
```
Comment (MineBartekSA) [inline src/bdscript/addEmoji.md]:
```suggestion
- `image URL`: The image of the new emoji. The link needs to be a valid image URL.
```

File: src/bdscript/addCmdReactions.md [hunk 2]
```diff
@@ -5,22 +5,21 @@ Adds reactions to the message that triggered the command.
...
+- `emojis`: The emoji(s) the bot reacts with. Separate emojis using `;`.
 
+> You can use Unicode emojis and emoji IDs, but not emoji names. For **emoji IDs**, the bot must be present in the server that the emoji originates from. 
+> 
 > List of unicode emojis: [😋 Get Emoji](https://getemoji.com)
  
-## Emoji IDs
+### How to get emoji ID?
+
+> This method requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) enabled!
```
Comment (MineBartekSA) [inline src/bdscript/addCmdReactions.md]:
```suggestion
> This method requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) to be enabled! 
```

File: src/bdscript/addCmdReactions.md [hunk 3]
```diff
@@ -5,22 +5,21 @@ Adds reactions to the message that triggered the command.
...
+
+> This method requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) enabled! 
+
 1. Type `\:TheEmojiName:`
 2. Send the message.
 3. Copy the ID it returns. (The emoji ID should be in this format: `<:emojiName:ID>`. If the emoji is animated, it should look like this: `<a:emojiName:ID>`)
 4. Input the emoji ID into `$addCmdReactions[]`. (e.g `$addCmdReactions[<:hollyDab:828628880629825546>]`)
 
-> 📝 This method requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) enabled! 
+> If you're still having issues, check [Troubleshooting](https://nilpointer-software.github.io/bdfd-wiki/nightly/resources/troubleshooting.html#the-bot-fails-to-add-reactions).
```
Comment (MineBartekSA) [inline src/bdscript/addCmdReactions.md]:
```suggestion
> If you're still having issues, check the [Troubleshooting page](https://nilpointer-software.github.io/bdfd-wiki/nightly/resources/troubleshooting.html#the-bot-fails-to-add-reactions).
```

File: src/bdscript/addField.md [hunk 4]
```diff
@@ -3,34 +3,34 @@ Adds a field to an embed.
 
 ## Usage
 ```
-$addField[name;value;(inline;index)]
+$addField[name;value;(inline?;index)]
 ```
-> 📝 You can create up to a maximum of 25 fields per embed.
+> You can create up to a maximum of 25 fields per embed.
```
Comment (MineBartekSA) [inline src/bdscript/addField.md]:
```suggestion
> You can create up to 25 fields per embed.
```

File: src/bdscript/addReactions.md [hunk 5]
```diff
@@ -5,22 +5,21 @@ Adds reactions to the bot's response.
...
+
+> This method requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) enabled! 
+
 1. Type `\:TheEmojiName:`
 2. Send the message.
 3. Copy the ID it returns. (The emoji ID should be in this format: `<:emojiName:ID>`. If the emoji is animated, it should look like this: `<a:emojiName:ID>`)
 4. Input the emoji ID into `$addReactions[]`. (e.g `$addReactions[<:hollyDab:828628880629825546>]`)
 
-> 🧙‍♂️ Note: If `\:TheEmojiName:` doesn't work, try `:-:TheEmojiName:`. This requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) enabled!
+> If you're still having issues, check [Troubleshooting](https://nilpointer-software.github.io/bdfd-wiki/nightly/resources/troubleshooting.html#the-bot-fails-to-add-reactions).
```
Comment (MineBartekSA) [inline src/bdscript/addReactions.md]:
```suggestion
> If you're still having issues, check the [Troubleshooting page](https://nilpointer-software.github.io/bdfd-wiki/nightly/resources/troubleshooting.html#the-bot-fails-to-add-reactions).
```

File: src/bdscript/addReactions.md [hunk 6]
```diff
@@ -5,22 +5,21 @@ Adds reactions to the bot's response.
...
+- `emojis`: The emoji(s) the bot reacts with. Separate emojis using `;`.
 
+> You can use Unicode emojis and emoji IDs, but not emoji names. For **emoji IDs**, the bot must be present in the server that the emoji originates from. 
+> 
 > List of unicode emojis: [😋 Get Emoji](https://getemoji.com)
  
-## Emoji IDs
+### How to get emoji ID?
+
+> This method requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) enabled!
```
Comment (MineBartekSA) [inline src/bdscript/addReactions.md]:
```suggestion
> This method requires [Developer Mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) to be enabled! 
```

File: src/bdscript/addButton.md [hunk 7]
```diff
@@ -3,14 +3,42 @@ Adds button to the response.
...
+- `label`: It's the text value visible on a button.
+- `style`: It's used to specify the button's background color. If the button has a link/url you **have to** set this to `link`. Check [this section](#button-style) for more details.
+- `disabled?`: If set to `yes` the button can't be pressed. Defaults as `no`. _(Optional)_
+- `emoji`: Adds an emoji inside the button. Emojis have to be either pasted as *unicode* or be in this format `<:emoji name:emoji ID>`. _(Optional)_
+- `message ID`: Adds a button to the provided message ID. It's important to note that provided message ID author **has to** be the bot. _(Optional)_
+
+>**Note:** Interactive buttons can't have duplicated `ID`'s in the same message. So for example, you can't have two buttons with the ID set to `test`.\
+\
+>**Note:** If `url` is used in `interaction ID or url` argument, it **has to** start with `http://` or `https://`
+
```
Comment (MineBartekSA) [inline src/bdscript/addButton.md]:
```suggestion 
## Parameters
- `new row?`: If set to `yes` the button will appear in a new row. If it's set to `no` the button will appear in the same row as a previous button.

    > 📝 A message can have a maximum of 25 buttons (5 rows of 5 buttons).

- `interaction ID/url`: Depending on the button type, you either set it to `interaction ID` which is then used in `$onInteraction[ID]` callback or `URL` if it's a link button.
- `label`: The text value visible on the button.
- `style`: It's used to specify the button's background color. If the button has a link/url you **have to** set the value to `link`. Check [this section](#button-style) for more details.
- `disabled?`: If set to `yes` the button can't be pressed. Defaults as `no`. _(Optional)_
- `emoji`: Adds an emoji inside the button. Emojis have to be either pasted as *unicode* or be in the following format `<:emoji name:emoji ID>`. _(Optional)_
- `message ID`: Adds a button to the provided message ID. It's important to note that provided message ID author **has to** be the bot. _(Optional)_

>**Note:** Interactive buttons can't have duplicated `ID`'s in the same message. So for example, you can't have two buttons with the ID set to `test`.\
\
>**Note:** If `url` is used in `interaction ID/url` argument, it **has to** start with `http://` or `https://`

## Button Style
Buttons can have different styles _(background colors)_.
Here, are all possible values for `style` function argument.
- primary
- secondary
- success
- danger
- link

> 📝 If `link` style is used, the button **won't send** any interactions!

```

File: src/bdscript/addButton.md [hunk 8]
```diff
@@ -3,14 +3,42 @@ Adds button to the response.
 
 ## Usage
 ```
-$addButton[New row;Button ID/URL;Label;Style;(Disabled;Emoji;Message ID)]
+$addButton[new row?;button ID/URL;label;style;(disabled?;emoji;message ID)]
```
Comment (MineBartekSA) [inline src/bdscript/addButton.md]:
```suggestion
$addButton[new row?;interaction ID/url;label;style;(disabled?;emoji;message ID)]
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]
Thank you for your hard work, but there are still changes I'd like you to make.

**Comment by Rainb0wKey** [comment]
Thanks!

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution

**Pull request merged**